CONTRIBUTING.md

```diff
@@ -33,14 +34,14 @@ If you set your `user.name` and `user.email` git configs, you can sign your comm
 
 Note: If your git config information is set properly then viewing the `git log` information for your commit will look something like this:
 
-```
-Author: Joe Smith <joe.smith@example.com>
-Date:   Thu Feb 2 11:41:15 2018 -0800
+    ```text
+    Author: Joe Smith <joe.smith@example.com>
+    Date:   Thu Feb 2 11:41:15 2018 -0800
 
-    Update README
+        Update README
 
-    Signed-off-by: Joe Smith <joe.smith@example.com>
-```
+        Signed-off-by: Joe Smith <joe.smith@example.com>
+    ```
```

Comment:
This block of code is not properly rendered

Reply:
Now it should be solved